std/algorithm/sorting.d

```diff
@@ -11,7 +11,7 @@ $(T2 completeSort,
         $(D completeSort(a, b)) leaves $(D a = [6, 10, 15]) and $(D b = [20,
         30, 40]).
         The range $(D a) must be sorted prior to the call, and as a result the
-        combination $(D $(REF chain, std,range)(a, b)) is sorted.)
+        combination $(REF chain, std,range)$(D (a, b)) is sorted.)
```

Comment:
NB: the added space:
![image](https://cloud.githubusercontent.com/assets/4370550/23589567/9f0e8284-01cf-11e7-97b1-3f42aec9bcd1.png)